Source/Data/Backend/package.json

```diff
@@ -30,8 +30,9 @@
         ],
         "ext": "ts"
     },
-    "dependencies": {
-        "@shared/backend": "1.0.0",
-        "@shared/specs": "1.0.0"
+    "devDependencies": {
+        "@apollo/client": "^3.3.11",
```

Comment:
Is this dependency needed?

Reply:
I will double check

Reply:
Removed and at least locally seems happy.